en-us/changed-options.md

```diff
@@ -67,6 +67,11 @@ Fabulously Optimized includes some mods that fix vanilla bugs.
 | [MC-235035](https://bugs.mojang.com/browse/MC-235035) | Sleeping in a custom dimension with "natural" set to false causes crash    | [Debugify](https://curseforge.com/minecraft/mc-mods/debugify)             |
 | [MC-237493](https://bugs.mojang.com/browse/MC-237493) | Telemetry cannot be disabled                                               | [Debugify](https://curseforge.com/minecraft/mc-mods/debugify)   |
 | [MC-249059](https://bugs.mojang.com/browse/MC-249059) | Loading terrain screen cannot close before 2 seconds have passed           | [Debugify](https://curseforge.com/minecraft/mc-mods/debugify)             |
+| [MC-249693](https://bugs.mojang.com/browse/MC-249693) | Chorus plant blocks explicitly define cullface arguments for faces which end up being obscured by other chorus plants. |  [MoreCulling](https://www.curseforge.com/minecraft/mc-mods/moreculling)  |
```

Comment:
Other titles don't have periods in the end, these shouldn't have either.

Reply:
He means: Remove the dots

Reply:
I made a PR in your fork fixing that